teaser/src/fpfh.cc

```diff
@@ -59,3 +59,5 @@ void teaser::FPFHEstimation::compute(pcl::PointCloud<pcl::FPFHSignature33>& outp
   fpfh_estimation_->compute(output_cloud);
 }
 void teaser::FPFHEstimation::setRadiusSearch(double r) { fpfh_estimation_->setRadiusSearch(r); }
+
+pcl::PointCloud<pcl::Normal> teaser::FPFHEstimation::getNormals() { return *normals_; }
```

Comment:
need to add a new line at the end of the file (I think clang format can do this automatically?)

Reply:
Yes, there's 
```
InsertFinalNewline: 'yes'
```
option. May I add that option to `.clang_format` and add an additional line for all files?

Reply:
But I've noticed that other files also have no blank line; could you tell me why the new line at the end is needed?

Reply:
Yes about clang format. If the other files do not have newline character at the end, then we should also add them. See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline